storage/innobase/xtrabackup/test/t/keyring_encryption.sh

```diff
@@ -0,0 +1,48 @@
+############################################################################
+# PXB-1961: xtrabackup fails with encryption='n' with Percona server
+############################################################################
+
+. inc/common.sh
+. inc/keyring_file.sh
+
+if ! is_debug_server  ; then
+       skip_test "Require debug server version"
+fi
```

Comment:
better to put this as function in common.sh as  have_debug() function 
and call that function here.

Reply:
I guess you can address these comments on 5.7 branch and merge to 8.0